atomic_reactor/util.py

```diff
@@ -1017,6 +1102,8 @@ def get_config_from_registry(image, registry, digest, insecure=False,
     tag = image.tag or 'latest'
     logger.debug('Image %s:%s has config:\n%s', context, tag, blob_config)
 
+    if return_config_digest:
+        return blob_config, config_digest
```

Comment:
Ick. Can't we make a new function `get_config_and_id_from_registry()`, and implement `get_config_from_registry()` using that?

Reply:
ok